content/topics/Analyze/causal-inference/introduction/synthetic-control-methods-intro.md

```diff
@@ -0,0 +1,207 @@
+                   margin_ipop = .02,sigf_ipop = 7,bound_ipop = 6 # optimizer options
+  ) %>%
+  
+  # Generate the synthetic control
+  generate_control()
+```
+{{% /codeblock %}}
+
+Once the synthetic control is generated, one can easily assess the fit by comparing the trends of the synthetic and observed time series. The idea is that the trends in the pre-intervention period should map closely onto one another. Let's verify that with a graph: 
+
+{{% codeblock %}}
+```R
+smoking_out %>% plot_trends()
+```
+{{% /codeblock %}}
+
+<p align = "center">
+<img src = "../images/1_synthetic.png" width="700">
+</p>
+
+As you can see, the pre-intervention trend between synthetic and observed cigarette sales is the same. 
+
+To capture the causal quantity (i.e. the difference between the observed and counterfactual), one can plot the differences using plot_differences(): 
+
+{{% codeblock %}}
+```R
+smoking_out %>% plot_differences()
+```
+{{% /codeblock %}}
+
+<p align = "center">
+<img src = "../images/2_synthetic.png" width="700">
+</p>
+
+Another useful way of evaluating the synthetic control is to look at how comparable the synthetic control is to the observed covariates of the treated unit by means of the following code: 
+
+{{% codeblock %}}
+```R
+smoking_out %>% grab_balance_table()
+```
+{{% /codeblock %}}
+ 
+## Advantages and limitations 
+
+According to Abadie (2021), the advantages of synthetic control methods are: 
+
+- SCMs provide a systematic way to estimate the effects of interventions when randomized controlled trials are not feasible. 
+- SCMs offer a transparent and data-driven method to construct the control group, which is based on observable characteristics and pre-treatment outcomes.
+- SCMs are flexible and can be applied in a wide range of contexts. 
+
+Possible disadvantages include: 
+
+- SCMs require extensive data on pre-treatment characteristics and outcomes for both treated and potential control units. 
+- The results of SCMs can be sensitive to the choice of predictor variables and the weighting scheme.
+There can be considerable variability in estimates depending on the specification of the synthetic control model.
+- Constructing synthetic controls can be computationally intensive, especially with large datasets or complex models.
+
+## Summary 
+
+{{% summary %}}
+
+- Synthetic control methods (SCMs) are statistical techniques used to estimate the effect of interventions by creating a synthetic version of the treatment group from a weighted combination of control units.
+- SCMs provide a transparent, data-driven approach for constructing control groups, making them suitable for contexts where randomized controlled trials are not feasible.
+- While SCMs offer flexibility and systematic estimation, they require extensive pre-treatment data and can be sensitive to model specifications and computationally intensive.
+
+{{% /summary %}}
+
+
+### References 
```

Comment:
write each reference in bullet points and hyperlink. For eg:

- [Using Synthetic Controls: Feasibility, Data Requirements, and Methodological Aspects](https://www.aeaweb.org/articles?id=10.1257/jel.20191450 DOI: 10.1257/jel.20191450)